### PR TITLE
test(tabs): added axe tree test for aria selected state

### PR DESCRIPTION
**Pull request by AdityaP7649**
## What I did

Added axe tree test for aria-selected state

### CODE REVIEW

File: elements/pf-tabs/test/pf-tabs.spec.ts [hunk 1]
```diff
@@ -241,6 +241,11 @@ describe('<pf-tabs>', function() {
             expect(second).to.have.attribute('active');
             expect(third).to.not.have.attribute('active');
           });
+
+          it('should specify the selected tab to assistive technology', async function() {
+            const snapshot = await a11ySnapshot();
+            expect(snapshot.children?.find(x => x.role === 'tabpanel')?.name).to.equal('tab-2');
```
Comment (bennypowers) [inline elements/pf-tabs/test/pf-tabs.spec.ts:247]: we can use ax helpers for this
```suggestion
            expect(await a11ySnapshot())
              .to.axContainQuery({ role: 'tabpanel', name: 'tab-2' });
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: elements/pf-tabs/test/pf-tabs.spec.ts (hunk 1 above)
Comment (AdityaP7649) [reply elements/pf-tabs/test/pf-tabs.spec.ts]: Hi @bennypowers,

Thank you for the review.
Updated the changes as mentioned.

Comment (bennypowers) [reply elements/pf-tabs/test/pf-tabs.spec.ts]:
```
 ✓  <pf-tabs> with three valid tab pairs when navigated by keyboard when focused on the first tab pressing ArrowRight should specify the selected tab to assistive technology [Chromium]
✓  <pf-tabs> with three valid tab pairs when navigated by keyboard when focused on the first tab pressing ArrowLeft should specify the selected tab to assistive technology [Chromium]
✓  <pf-tabs> with three valid tab pairs when navigated by keyboard when focused on the first tab pressing ArrowLeft then pressing ArrowRight should specify the selected tab to assistive technology [Chromium]
```

Looks good!

**Review by bennypowers — APPROVED** [review]
Lovely tests: gets merged!

**Pull request merged**